java/bulk-export/bulk-export-task-execution/src/main/java/sleeper/bulkexport/taskexecution/ECSBulkExportTaskRunner.java

```diff
@@ -107,21 +128,68 @@ public static void runECSBulkExportTaskRunner(AmazonSQS sqsClient, InstancePrope
         exportQueueHandler.receiveMessage().ifPresent(messageHandle -> {
             try {
                 BulkExportLeafPartitionQuery exportTask = messageHandle.getJob();
-                LOGGER.info("Received leaf partition bulk export job: {}", exportTask);
-                exportTask.getFiles().forEach(inputFile -> {
-                    LOGGER.info("Input file: {}", inputFile);
-                });
-                String partitionId = exportTask.getLeafPartitionId();
-                LOGGER.info("Partition ID: {}", partitionId);
-                String tableId = exportTask.getTableId();
-                LOGGER.info("Table ID: {}", tableId);
+                LOGGER.info("Received bulk export job for table ID: {}, partition ID: {}", exportTask.getTableId(), exportTask.getLeafPartitionId());
+                LOGGER.debug("Export task details: {}", exportTask);
```

Comment:
I think calling the query a task is a bit confusing. A task is an ECS concept, it's the container that this code runs in.

In the line above we called it a bulk export job, can we use the same language throughout?